### ISSUE TITLE
Transformers QA Online Demo is not working

### ISSUE THREAD

**Issue by gavishpoddar** [huggingface/transformers#10976]
Transformers QA Online Demo is not working: https://huggingface.co/qa/

I am trying to recreate ELI5 but unable to find enough information @yjernite Can you Please help. 

Please let me know if I can help 

Thanks

**Comment by yjernite** [comment]
I re-launched the app.

The app itself doesn't have much information to help recreate the system though (and is not designed for heavy use :) )

I'd recommend reading through the blog post instead: https://yjernite.github.io/lfqa.html

**Comment by gavishpoddar** [comment]
Thank you so much @yjernite
